docs/operate/relayer/run-relayer.mdx

```diff
@@ -47,21 +47,69 @@ See the [Agent Keys](../set-up-agent-keys.mdx) page to set up your Hexadecimal o
 
 ### Configuration
 
+#### RPC Configuration
+
+Hyperlane Validators and Relayers can use **multiple RPC URLs** to improve reliability and redundancy. The setup varies based on the chain type.
+
+<Tabs groupId="rpc-config">
+  
+<TabItem value="evm" label="EVM Chains">
+
+EVM-based chains support configuring multiple RPC endpoints for redundancy and can specify how they are used.
+
+- **Configure multiple RPCs:** Use [`customRpcUrls`](../config-reference.mdx#chainschain_namecustomrpcurls).
+- **RPC Selection Mode ([`rpcConsensusType`](../config-reference.mdx#chainschain_namerpcconsensustype))**:
+  - **Fallback**: The agent tries the first URL and switches to the next if needed.
+  - **Quorum**: Requires a majority of the URLs to agree with the exception of submitting transactions; it will automatically coordinate the "latest" block if not specified to reduce sync errors.
```

Comment:
> it will automatically coordinate the "latest" block if not specified to reduce sync errors.

found this a little confusing - imo this could be removed